shell/platform/fuchsia/dart_runner/tests/startup_integration_test/dart_jit_runner/dart_jit_echo_server/BUILD.gn

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I don't think this file should exist, maybe merge conflict?

Reply:
It wouldn't let me push the change without formatting it. Not sure how to get around it.

Reply:
If you delete it, the linter still complains? What.. 😭

Reply:
Oh wait, you mean the entire file just doesn't belong in flutter/engine ?